### PR TITLE
Support zooming in/out

### LINKED ISSUE #59: Support zooming in / out
* `Ctrl` + `Scroll` to zoom in / out
* `Ctrl` + `+` to zoom in
* `Ctrl` + `-` to zoom out
* `Ctrl` + `0` to go back to default zoom level

### PR DESCRIPTION
**Pull request by ryonakano**
2020-08-25 08 55 15 の画面録画

Fixes #59

Use `Ctrl + Scroll` or the scale at the bottom of the window to zoom in/out.

### CODE REVIEW

File: data/spreadsheet.gschema.xml [hunk 1]
```diff
@@ -21,5 +21,10 @@
       <summary>Recent files</summary>
       <description>Files that are most recently opened or saved</description>
     </key>
+    <key name="zoom-level" type="i">
+      <default>100</default>
+      <summary>Zoom lebel</summary>
```
Comment (elegaanz) [inline data/spreadsheet.gschema.xml:26]: Typo here: `level`

File: src/Widgets/ActionBar.vala [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        }
+    }
+
+    public string zoom_level_text {
+        owned get {
+            return "%i %%".printf (zoom_level);
+        }
+    }
+
+    public ActionBar () {
```
Comment (elegaanz) [inline src/Widgets/ActionBar.vala:22]: Is this constructor really needed? Doesn't the compiler adds it for you?

Comment (ryonakano) [reply src/Widgets/ActionBar.vala:22]: You're right. I added this but it's not really necessary.

**Review by elegaanz — APPROVED** [review]
Perfect! Thank you!

**Pull request merged**